CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## Versions
 
+## x.x.x
+    * Update the example app to demonstrate iOS' Max Consent Flow.
```

Comment:
The CHANGELOG is only for the plugin itself, the example app is not part of the plugin - no need to document anything there.
```suggestion
```

Reply:
ok, got it